packages/react-jsx-highcharts-gantt/src/index.js

```diff
@@ -0,0 +1,24 @@
+import { withSeriesType } from 'react-jsx-highcharts';
+
+// TODO: Add more common modules that Highcarts Gantt support
+export {
+  Chart,
+  Title,
+  Subtitle,
+  Tooltip,
+} from 'react-jsx-highcharts';
+
+
+// Charts
+export { default as HighchartsGanttChart } from './components/HighchartsGanttChart';
+
+// Graph Parts
+
+export { default as XAxis } from './components/XAxis'; // TODO: ?
+export { default as YAxis } from './components/YAxis'; // TODO: ?
```

Comment:
These will be exactly the same as the XAxis and YAxis in `react-jsx-highmaps` (we can clean that up at a later date)